README.md

```diff
@@ -69,13 +69,13 @@ package manager to search for string 'screengrab'.
 
 ## LICENSE
 
-Screengrab is licensed under the GPL v2. See file LICENSE in docs directory for more information
+ScreenBrab is licensed GPLv2+. The LICENSE file in the docs directory has more info.
```

Comment:
IMO the first is better worded
Typo in ScreenBrab

Reply:
I think so too, but that is a more concerted effort.
I failed at this, so someone else should be in charge of figuring out how many places "ScreeGrab" is used.